5.findBugCommit.js

```diff
@@ -24,7 +24,11 @@
 //  }  
 
 const identifyFirstBadCommit = (commits) => {
-  
+    // quicksort???
+    if(commits.length <= 1) return arr
+   let output = {}
+
+   return output
 }
```

Comment:
This problem requires a binary search so that it runs in Log(n). The data is already sorted. Please revisit.